.github/release-drafter.yml

```diff
@@ -0,0 +1,21 @@
+name-template: 'v$NEXT_PATCH_VERSION 🌈'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
```

Comment:
Only use labels that are available (https://github.com/fossasia/neurolab-android/labels)